assets/src/scripts/store/index.js

```diff
@@ -289,6 +325,23 @@ export const Actions = {
             period
         };
     },
+    setRequestedDates(startTime, endTime) {
+        return {
+            type: 'SET_REQUESTED_DATES',
```

Comment:
Ditto with the name here, maybe SET_CUSTOM_DATE_RANGE. Of course, I see this bubbles into other places but in my mine all of the date ranges are requested, so it's a little confusing more me. YMMV.

Reply:
Done.